docs/source/development.rst

```diff
@@ -15,6 +15,9 @@ dependencies for local development.
    $ source env/bin/activate
    (env)$ python -m pip install -e .
 
+**Note:** If you are using macOS, you need to have PostgreSQL installed. You can
```

Comment:
Two things:

1. Sphinx and restructuredText has a [`.. node::` directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html?highlight=note#directive-note) that should be used instead:
   ```rst
   .. node::

      If you are using macOS, you need to have PostgreSQL installed. You can
      install it via Homebrew_ (``brew install postgresql``) or follow the
      `PostgreSQL docs`_.
   ```
2. Can you move the node above the `code-block` directive. It's often nicer to see such notes _before_ running into trouble.